lib/Service/ProvisioningService.php

```diff
@@ -149,6 +109,17 @@ public function provisionUser(string $tokenUserId, int $providerId, object $idTo
 		if ($existingLocalUser !== null) {
 			$user = $existingLocalUser;
 		} else {
+			// if disable_account_creation is true, user_oidc should not manage user
+			// - no user creation
+			// - do not use the existing users managed by the user_oidc backend
```

Comment:
Hum but what if user_oidc created some users and then I enable the new option, these users disappear? Or at least cannot login anymore?

Reply:
They can't login... I took the easy way. I can change that behaviour. Do you think it makes more sense to let them in?